src/pilot/discover_nodes/discover_nodes/dracclient/client.py

```diff
@@ -77,7 +77,7 @@ def __init__(self,
                                          path,
                                          protocol)
         self._job_mgmt = job.JobManagement(self.client)
-        self._idrac_cfg = idrac_card.iDRACCardConfiguration(self.client)
+        self._idrac_con = idrac_card.iDRACCardConfiguration(self.client)
```

Comment:
Now that `config_idrac.py` uses upstream's `reset_idrac()`, does `discover_nodes.resources.idrac_card` add any value? If not, this member variable and that module could be removed now **and** resolve a puzzling situation, rather than perpetuate it.

Thoughts?

Reply:
Richard, Earlier I tried removing downstream drac client from config_idrac.py but faced multiple dependency issues, So we decided to go task by task which may need to add some temporary changes to local downstream drac client but eventually, all the repeated and unused code will be removed.
This way we avoid complications in code and it will be easy to review the code.

I currently facing some issues regarding dependencies of set_idrac_settings method.
Getting 
"InvalidParameterValue: Unknown iDRAC Card attributes found: set(['A', 'C', 'b', 'E', 'D', 'i', 'm', '.', '1', 'R', 'e', 'd'])"

Reply:
Modified this PR with requested changes, Please review